src/lob.dotnet/Model/Letter.cs

```diff
@@ -764,7 +764,7 @@ public IEnumerable<System.ComponentModel.DataAnnotations.ValidationResult> Valid
             }
 
             // Url (string) pattern
-            Regex regexUrl = new Regex(@"^https:\/\/(lob-assets|lob-assets-staging)\\.com\/(letters|postcards|bank-accounts|checks|self-mailers|cards)\/[a-z]{3,4}_[a-z0-9]{15,16}(\\.pdf|_thumb_[a-z]+_[0-9]+\\.png)\\?(version=[a-z0-9-]*&)?expires=[0-9]{10}&signature=[a-zA-Z0-9-_]+$", RegexOptions.CultureInvariant);
+            Regex regexUrl = new Regex(@"^https:\/\/lob-assets.com\/(letters|postcards|bank-accounts|checks|self-mailers|cards)\/[a-z]{3,4}_[a-z0-9]{15,16}('|_signature)(.pdf|_thumb_[a-z]+_[0-9]+.png|.png)?(version=[a-z0-9]*&)expires=[0-9]{10}&signature=[a-zA-Z0-9-_]+", RegexOptions.CultureInvariant);
```

Comment:
What is this change doing?

Reply:
It should have been omitted from the changes, let me fix that